lmdeploy/tokenizer.py

```diff
@@ -220,6 +221,32 @@ def __init__(self, model_dir: str):
         self.max_indexes_num = 5
         self.token2id = {}
 
+    def _check_transformers_version(self, model_dir: str):
+        import transformers
+        from packaging import version
+        from transformers.models.auto.configuration_auto import \
+            CONFIG_MAPPING_NAMES
+
+        from lmdeploy.utils import get_hf_config_content
+
+        logger = get_logger('lmdeploy')
+
+        current_transformers_version = version.parse(transformers.__version__)
+        cfg = get_hf_config_content(model_dir, trust_remote_code=True)
```

Comment:
please use `get_model_arch` instead.
`get_hf_config_content` is going to be removed since its functionality is already included in `get_model_arch`.

Reply:
ok